docs/proposals/PROPOSAL_E.md

```diff
@@ -52,57 +51,65 @@ These annotations should be considered from the current config schema:
 
 Existing `org.opencontainers.image.*` annotations should be reviewed to consider more generic names (e.g. replacing `image` with `manifest` or `artifact`).
 
-#### Image Spec
+#### Artifact Spec
 
-Extend the Image Manifest with a reference field (existing registries should ignore this per OCI's extensibility requirement):
+Create a new artifact media type to support future use cases where a separate config blob, and an ordered list of blobs, may not match an artifact's use case:
 
 ```jsonc
 {
   "schemaVersion": 2,
-  "mediaType": "application/vnd.oci.image.manifest.v1+json",
-  "config": { ... },
-  "layers": [ ... ],
-  "reference": { // new field
-    "mediaType": "application/vnd.oci.image.manifest.v1+json", // any manifest media type
-    "size": 1234,
-    "digest": "sha256:a1a1a1...",
-    "annotations": []
+  "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
```

Comment:
There was a good discussion of considering if should bring in artifactType as a first class field. 
This comment is non-blocking for merging this PR. So please do consider if if retType group and recommend a refType/artifactType,